mscore/scoretab.cpp

```diff
@@ -146,7 +146,7 @@ ScoreView* ScoreTab::view(int n) const
 QSplitter* ScoreTab::viewSplitter(int n) const
       {
       const TabScoreView* tsv = tabScoreView(n);
-      IF_ASSERT_FAILED(tsv) {
+      if(!tsv) {
```

Comment:
This should have had a space after the `if`, but I guess I'm too late.

Reply:
Yes, on both ;-)